hack/local-up-cluster.sh

```diff
@@ -681,6 +716,9 @@ EOF
 fi
 }
 
+# let's check that etcd is in path before moving on
```

Comment:
I like that idea, my only concern was starting etcd too early in the process while i just wanted to ensure that the process even existed, additionally that would sort of break the nice logic starting in line 748 where services are all started together.  

Would you be ok with me instead refactoring the logic found in kube::etcd::start so that a new function is created kube::etcd::validate that is called in start, so that the validation check is at least in one place?